gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -109,12 +109,20 @@ http:
 # For more information about MongoDB configuration, please have a look to:
 # - http://mongodb.github.io/mongo-java-driver/4.1/apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.Builder.html
 management:
+# Override configuration for retryable exceptions (transient) such as transaction conflict.
+#  transaction:
+#    retry:
+#      maxAttempts: 5
+#      delayMs: 100
+#      maxDelayMs: 500
+
```

Comment:
Should we also add this in the gateway gravitee.yml?

Reply:
No need because the annotation is only on management services.